src/main/java/seedu/address/logic/commands/FilterMeetingCommand.java

```diff
@@ -0,0 +1,61 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.logic.Messages;
+import seedu.address.model.Model;
+import seedu.address.model.meeting.AttendeeContainsKeywordsPredicate;
+import seedu.address.model.meeting.LocationContainsKeywordsPredicate;
+import seedu.address.model.meeting.MeetingTimeContainsPredicate;
+import seedu.address.model.meeting.TitleContainsKeywordsPredicate;
+
+/**
+ * Finds and lists all persons in address book whose name contains any of the argument keywords.
+ * Keyword matching is case insensitive.
+ */
+public class FilterMeetingCommand extends Command {
+
+    public static final String COMMAND_WORD = "filterm";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all meetings whose titles contain any of "
+            + "the specified keywords (case-insensitive) and displays them as a list with index numbers.\n"
+            + "Parameters: KEYWORD [MORE_KEYWORDS]...\n"
+            + "Example: " + COMMAND_WORD + " CS2103T Meeting";
+
+    private final TitleContainsKeywordsPredicate predicate;
+
+    public FilterMeetingCommand(TitleContainsKeywordsPredicate predicate, LocationContainsKeywordsPredicate predicate1, MeetingTimeContainsPredicate predicate2, AttendeeContainsKeywordsPredicate predicate3, ) {
```

Comment:
I'm assuming you will update this to use predicate1/2/3 since they do not do anything for now